src/main/webapp/resources/js/pages/analysis/components/Analysis.jsx

```diff
@@ -114,7 +147,7 @@ export default function Analysis() {
               analysisContext.analysisType.type === "MLST_MENTALIST" ? (
                 <TabPane
                   tab={getI18N("Analysis.phylogeneticTree")}
-                  key="PHYLOGENOMICS"
+                  key="phylogenomics"
                   className="t-analysis-tab-phylogenetic"
                 >
                   <AnalysisPhylogeneticTree />
```

Comment:
I am a little concerned about this.  I don't know how we should handle it, but I feel that this tab should be generated by some server variables left on the page.

Reply:
Yup we are going to have to do this a different way. This, the bio_hansel, and sistr pages are placeholders for now